controllers/openshift_controller.go

```diff
@@ -174,6 +177,20 @@ func (r *KataConfigOpenShiftReconciler) processDaemonsetForCR(operation DaemonOp
 									MountPath: "/host",
 								},
 							},
+							Env: []corev1.EnvVar{
```

Comment:
qq: what happens if the config map value changes at runtime?

Reply:
It's ignored. It's evaluated only once before the daemon downloads the payload image.

Reply:
If that's the desired behavior then it's fine. I was asking because if that is for the purpose of development, a developer would probably want to test with multiple images. To do so, they would be required to restart the controller.